.github/workflows/build-test.yml

```diff
@@ -0,0 +1,43 @@
+name: Snyk Vulnerability Scan
+
+on:
+  push:
+    branches:
+      - main  
+  pull_request:
+    branches:
+      - main 
+
+jobs:
+  build-and-scan:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '7.0'
+
+      - name: Restore dependencies
+        run: dotnet restore
+
+      - name: Install Snyk CLI
+        run: npm install -g snyk
+
+      - name: Authenticate Snyk
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        run: snyk auth $SNYK_TOKEN
+
+      - name: Run Snyk Test
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        run: snyk test --file=ConstantReminderApi.sln --severity-threshold=high
+
+      - name: Monitor Project in Snyk
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
did you figure out how to input the snyk token?

Reply:
in the repo its under settings somewhere

Reply:
i have a snyk private token that i used locally to run the snyk test.

Reply:
@funkel1989 I do not have access to add the secret key to this repo because I do not have admin rights. I do not see the settings option. However I believe I have admin rights to the UI so I will add it there. Once I get permission to do so for this repo, I will add it within the settings.